dev/com.ibm.ws.jaxws.2.2.webcontainer_fat_extended/publish/files/AddedElementHandlerChainTestServer/servicename-false-global-true-server.xml

```diff
@@ -0,0 +1,17 @@
+<server>
+    <featureManager>
+        <feature>jsp-2.2</feature>
+        <feature>jaxws-2.2</feature>
+    </featureManager>
+    
+   
+
+    <include location="../fatTestPorts.xml"/>
+    
+    <webServiceClient serviceName="SayHelloServiceWithHandler" ignoreUnexpectedElements="false"/>
+    
+    <webServiceClient ignoreUnexpectedElements="false"/>
```

Comment:
Name of the server configuration is servicename-false-global-true-server.xml. Either I would expect one of the value to be true, or name of the server configuration should be different.